css/sections/hero.css

```
@@ -0,0 +1,438 @@
+		100% {
+			border-color: transparent;
+		}
+	}
+
+	@keyframes hover-border {
+		0% {
+			background-color: #0000;
+		}
+		100% {
+			background-color: var(--paper-background, var(--paper-background));
+		}
```

Stylelint (17.2.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)